enhancements/microshift/microshift-default-csi-plugin.md

```diff
@@ -0,0 +1,345 @@
+    kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: my-lv-pvc
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 1G
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: my-pod
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+        command: ["/usr/bin/sh", "-c"]
+        args: ["sleep", "1h"]
+        volumeMounts:
+        - mountPath: /mnt
+          name: my-volume
+      volumes:
+        - name: my-volume
+          persistentVolumeClaim:
+            claimName: my-lv-pvc
+    EOF
+    ```
+
+**Maintainer Driven Rebase**
+
+Because MicroShift is based on OCP, it must ensure parity between its cluster components and those of OCP, by version.
+OCP records cluster component image digests in the ocp-release image, which is versioned according to its corresponding
+OCP version.  ODF maintains a similar release image called ocs-release, which maps ODF-LVM image digests to OCP versions.
+The workflow is automated and documented 
+[here](https://github.com/openshift/microshift/blob/9dfddbd6801098c96bbd7db887c2844cad177c69/docs/rebase.md).
+
+#### Variation
+
+**Start MicroShift with a user-provided ODF-LVM configuration**
+
+> After the device owner has logged into the device ... 
+
+1. The device owner will create a file called `/etc/microshift/lvmd.yaml`
+1. The device owner will define the lvmd config, according to its 
+[documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md)
+
+   _For example:_
+   
+    ```shell
+    $ cat <<EOF > /etc/microshift/lvmd.yaml
+    socket-name: 
+    deviceClasses:
+    - default: true
+      name: "user-provided-config"
+      volume-group: rhel
+      spare-gb: 10 # in GB
+    socket-name: /run/lvmd/lvmd.sock # socket-name must always be /run/lvmd/lvmd.sock
+    ```
+
+1. The device owner will start the microshift service, and proceed to validate the platform as described in **Validate 
+ODF-LVM function**
+1. The device owner will validate the system, following workflow **Validate ODF-LVM function**
+
+**Start MicroShift with a *malformed* user-provided ODF-LVM configuration**
+
+> After the device owner has logged into the device ...
+
+1. The device owner creates an incorrect lvmd config file under /etc/microshift/lvmd.config
+
+    _For example:_
+    ```shell
+    [root@ushift ~]# cat <<EOF > /etc/microshift/lvmd.yaml
+    socket-name: 
+    deviceClasses:
+    - default: true
+      name: "user-provided-config"
+      volume-group: non_existent_volume_group
+      spare-gb: 10 # in GB
+    socket-name: /run/lvmd/lvmd.sock
+    ```
+**Correct a _malformed_ user-provided ODF-LVM configuration and restart MicroShift**
+
+> After the device owner has logged into the device ...
+
+1. The device owner will edit the /etc/microshift/lvmd.yaml file to correct the error
+1. The device owner will restart MicroShift: `systemctl restart microshift`
```

Comment:
@nbalacha - wouldn't it be enough to restart the lvmd pod for the lvmd.yaml changes to take effect?